modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,668 @@
+= Production Readiness Checklist 
+
+Before running production workloads on Redpanda, work through the readiness checklists provided here to ensure that your cluster is deployed and configured according to Redpanda Data recommendations. The checklists help ensure that you are set up for success, and that you are ready to run mission-critical workloads in the largest of enterprises.
+
+Redpanda provides multiple resources for deploying a production cluster, and recommends that you utilize the xref:deploy:deployment-option/self-hosted/manual/production/production-deployment-automation.adoc[automated deployment instructions] in the Redpanda Ansible collection. If you cannot deploy with Ansible, use the xref:deploy:deployment-option/self-hosted/manual/production/production-deployment.adoc[manual deployment instructions].
+
+== Level 1 production readiness
+
+The Level 1 readiness checklist helps you to confirm that:
+
+- All required defaults and configuration items are specified.
+- You have the optimal setup of the hardware.
+- Security is enabled.
+- You are set up to run in production.
+
+[[redpanda-license]]
+=== Load Redpanda license
+
+Ensure that the xref:get-started:licenses.adoc[Redpanda License] has been loaded into the cluster configuration. This is required to enable xref:get-started:licenses.adoc#redpanda-enterprise-edition[Enterprise] features. 
```

Comment:
```suggestion
Check that the xref:get-started:licenses.adoc[Redpanda License] has been loaded into the cluster configuration. This is required to enable xref:get-started:licenses.adoc#redpanda-enterprise-edition[Enterprise] features. 
```

Reply:
Checklist items started with "ensure", "confirm", "check", and sometimes "check to ensure". It's a small thing, but I added suggestions to start them with "Check".